admin/resources/js/components/poolCandidate/UpdatePoolCandidate.tsx

```diff
@@ -81,9 +90,7 @@ export const UpdatePoolCandidateForm: React.FunctionComponent<UpdatePoolCandidat
   }) => {
     const intl = useIntl();
     const locale = getLocale(intl);
-    const dataToFormValues = (
-      data: PoolCandidate | UpdatePoolCandidateMutation["updatePoolCandidate"],
-    ): FormValues => ({
+    const dataToFormValues = (data: PoolCandidate): FormValues => ({
```

Comment:
I'd like to have data as type PoolCandidate | ( UpdatePoolCandidateMutation["updatePoolCandidate"] OR UpdateUserMutation["updateUser"]) but I couldn't figure out how to express that.

Reply:
I tried some variations but didn't result in anything 😢. I'll take a stab again in a bit.

Reply:
I think this should be fine to merge for now.